### ISSUE TITLE
Gifs in the docs are hard to follow

### ISSUE THREAD

**Issue by andybak** [LuaLS/LuaLS.github.io#9]
They move so fast I can't really take in what they are showing.

Ideally there would be some way to pause these. (I've looked around for a Firefox extension to do this but couldn't find one).

But yeah - Animated GIFs aren't really ideal for this kind of thing.

**Comment by carsakiller** [comment]
I thought Firefox (and maybe others) allowed you to right-click a GIF and reveal playback controls, but I must be thinking of `<video/>` elements. I'll look at converting the GIFs to videos.

**Comment by carsakiller** [comment]
I meant to replace the GIFs for a while, glad it is done now! Hopefully they are easier to follow along now that you can pause, rewind, and change the playback speed on them.
